htdocs/bootstrap/css/custom-css.css

```diff
@@ -446,3 +444,17 @@ a.btn.btn-primary {
 .sweet-alert p {
   word-wrap: break-word;
 }
+/* Loader customizations */
+
+.loader {
+    margin: auto;
+    border: 8px solid #FFFFFF; /* White */
+    border-top: 8px solid #3498db; /* Blue */
+    border-radius: 50%;
+    animation: spin 0.8s linear infinite;
+}
+
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
```

Comment:
Not sure how this snippet of code made it in, but it's actually part of `React Loader` PR (#3717), which is not yet merged.

Reply:
I was taking a look at #3717 before this PR and i'll remove as I didn't mean for this to be included.